.github/workflows/non-release.yaml

```
@@ -0,0 +1,24 @@
+name: Build
```

Shouldn't this be called "NonRelease"?

```suggestion
name: NonRelease
```

---

yes. I split (as documented [here](https://gardener.github.io/cc-utils/github_actions.html#migration-from-concourse-pipelines)) the github-actions-pipelines like so:

- build.yaml # contains re-usable build (for OCI-Images, helmcharts, tests, ..)
- non-release.yaml # contains triggers (pullrequests and head-updates)
- release.yaml # contains manual trigger for release-jobs (in concourse, this was by convention called "manual-release")

You are completely free to choose any name, and also different layouts. The Proposal I raised is equivalent to the existing Concourse-Pipelines, while at the same time reducing redundancies as much as possible.

So what should we do now w.r.t. naming?

---

how should we proceed with this naming-topic? As mentioned, I don't mind either way - you will need to choose (on another note, you can obviously rename at a later point in time)